release notes/v0.49.0.md

```diff
@@ -17,6 +18,33 @@ k6 `v0.49.0` is here 🎉! This release includes:
 
 ## New features
 
+### Web Dashboard
+
+The new web dashboard brings real-time visualization to load testing. This feature allows users to monitor test progress and analyze
+results dynamically, enhancing the overall testing experience.
+
+#### Real-time test results
+
+Activate this feature using the environment variable K6_WEB_DASHBOARD=true. For this initial release, the dashboard is not enabled
+by default to allow users to opt into this new experience as it continues to evolve.
+
+```bash
+K6_WEB_DASHBOARD=true k6 run script.js
+```
+
+Once enabled and the test script is running, navigate to [http://localhost:6565](http://localhost:6565) in your web browser to access the dashboard.
+
+![k6 Web Dashboard Overview](https://github.com/grafana/xk6-dashboard/blob/master/screenshot/k6-dashboard-overview-light.png?raw=true)
+
+#### Test report
+
+The web dashboard also offers an HTML test report for detailed analysis, enabling easy sharing and downloading capabilities for
+collaboration.
+
+To access and download the report, click on the `Report` button located in the top right corner of the dashboard.
+
+![HTML Report Generated by k6 Dashboard](https://github.com/grafana/xk6-dashboard/blob/master/screenshot/k6-dashboard-html-report.png?raw=true)
```

Comment:
TBH, I'm wondering if both images should be embedded. For instance, the second image could remain a link, and customers could open and see a complete example of the report in a separate tab/window.

Reply:
I was wondering the same, I don't think I have a strong take on either way, although I find that this feature is very visual, and giving a visual clue is a good call to action. Maybe we keep the first image and keep the second as link instead?

Reply:
> Maybe we keep the first image and keep the second as link instead?

Yes, that's exactly what I was wondering, too :+1: It's essential to attract the attention of our customers, but I believe if both images are embedded, it could overload attention. 

So :+1: for keeping the first image as it is and moving the second to a link, something like _an example of the report you could find here_ or a better sugegstion

Reply:
Perfect wilco 👍🏻